data/en/settimezone.json

```diff
@@ -3,14 +3,20 @@
 	"type": "function",
 	"syntax": "setTimezone(timezone)",
 	"returns": "string",
-	"related": [],
+	"related": ["getTimezone","getTimezoneInfo","setLocale"],
 	"description": "Change the timezone defintion for the current request",
 	"params": [
 		{"name": "timezone", "description": "", "required": true, "default": "", "type": "string", "values": []}
 	],
 	"engines": {
 		"lucee": {"minimum_version": "", "notes": "", "docs": "http://docs.lucee.org/reference/functions/settimezone.html"}
 	},
-	"examples": [],
-	"links": []
-}
\ No newline at end of file
+	"links": [],
+	"examples": [{
+		"title": "Change Timezone from UTC to CET",
+		"description": "",
+		"code": "writeOutput(getTimezone());\nsetTimezone(\"CET\");\nwriteOutput(\"→ \"&getTimezone());",
```

Comment:
Hmm this example is not working for some reason on travis:

gettimezone.json example result is:Etc/UTC but evaluated to:EXCEPTION in example 1: Can't cast Object type [sun.util.calendar.ZoneInfo] to a value of type [timezone]. Expected [Etc/UTC] Actual [EXCEPTION in example 1: Can't cast Object type [sun.util.calendar.ZoneInfo] to a value of type [timezone]]

It seams to work on trycf - I wonder if a bug in the specific version of Lucee we are testing with.

Reply:
It seems that it returns the result of `getTimezoneInfo()` which is a structure instead of `getTimezone()` which is a string...

Reply:
Looks like a bug. Works here as well for example: http://tpcg.io/dhnnU7

Reply:
But it works for `getTimezone()`'s example, though...

Reply:
@pfreitag So what should we do? Nevertheless everything else works well as you can see.